eventstore/eventstore.go

```diff
@@ -39,32 +46,129 @@ type EventStore struct {
 // Open opens the event store. It should be closed later with the
 // Close() method.
 func Open(fileName string) (*EventStore, error) {
-	db, err := bolt.Open(fileName, 0600, nil)
+	db, err := bolt.Open(fileName, 0600, &bolt.Options{Timeout: openTimeout})
 	if err != nil {
 		return nil, err
 	}
 
+	log.Println("getting events to migrate from old bucket")
+	eventsToMigrate, oldEventTimes, err := getEventsToMigate(db)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Stop the migration here if there's nothing to migrate. There's no need to log anything or do anything else if there's nothing to migrate.